autoreduce_qp/queue_processor/reduction/process_manager.py

```diff
@@ -27,33 +25,89 @@ def __init__(self, message: Message, run_name: str) -> None:
     def run(self) -> Message:
         """Run the reduction subprocess."""
         try:
+            # Create a temp directory to store reduction result
+            # Volume bind for the container to write to and host to read from
+            temp_dir = tempfile.TemporaryDirectory()  # pylint: disable=consider-using-with
+            os.chmod(temp_dir.name, 0o777)
```

Comment:
I think this chmod needs to always be done, but the rest can be done only `if not AUTOREDUCTION_PRODUCTION` and that can be handled in Ansible